docs/demo/gaiden-css/components/card/custom.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+  <style>
+    body {
+      background-color: #eff4fa;
+    }
+
+  </style>
+</head>
+
+<body>
+  <div class="container">
+    <div class="col-small-6">
```

Comment:
@MarcoBrunoBR Nice tip! You are right, we don't need the extra wrapper. I will change this only for this demo. Maybe another scenario could arise where we could need one. 👍